ansible_collections/arista/avd/roles/eos_designs/python_modules/network_services/router_bgp.py

```diff
@@ -17,6 +19,8 @@ class RouterBgpMixin(UtilsMixin):
     Class should only be used as Mixin to a AvdStructuredConfig class
     """
 
+    _configure_bgp_mlag_peer_group: bool
```

Comment:
do you still need this now that it is a cached property?

Reply:
removed